edge/pkg/metamanager/metaserver/handlerfactory/handler.go

```diff
@@ -219,7 +219,7 @@ func limitedReadBody(req *http.Request, limit int64) ([]byte, error) {
 		return nil, err
 	}
 	if lr.N <= 0 {
-		return nil, errors.NewRequestEntityTooLargeError(fmt.Sprintf("limit is %d", limit))
+		return nil, errors.NewRequestEntityTooLargeError(fmt.Sprintf("Limit is %d", limit))
```

Comment:
This will be referenced by the `errors.NewRequestEntityTooLargeError` method, lowercase is recommended.
Other places too.